README.md

```diff
@@ -109,55 +162,72 @@ trigger:
-|`bridge_download_version`| Provide bridge version.<br/> If provided, the specified version of Synopsys Bridge is downloaded and configured.              | Optional     |
-| `include_diagnostics`      | All diagnostics files are available to download when `true` passed.<br/> Azure DevOps no longer supports per-pipeline retention rules. The only way to configure retention policies for YAML and classic pipelines is through the project settings.<br/> Refer the given documentation for more details: <br/> https://learn.microsoft.com/en-us/azure/devops/pipelines/policies/retention?view=azure-devops&tabs=yaml#set-run-retention-policies               | Optional     |
-
-Note - If `bridge_download_version` or `bridge_download_url` is not provided, Synopsys Security Scan downloads and configure the latest version of Bridge.
- 
-# Synopsys Bridge Setup
-
-- The latest version of Synopsys Bridge is available at: [Synopsys-Bridge](https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/synopsys/integration/synopsys-bridge/) 
-- The most common way to set up Synopsys Bridge is to configure the agent to download the small CLI utility automatically run at the right stage of your pipeline.
-- The latest version of Synopsys Bridge is downloaded by default.
-
-## Manual Synopsys Bridge
-
-If you are unable to download the Synopsys Bridge from our internet-hosted repository, or have been directed by support or services to use a custom version of the Synopsys Bridge, you can either specify a custom URL or pre-configure your agent to include the Synopsys Bridge. In this latter case, you would specify the `synopsys_bridge_path` parameter to specify the location of the directory in which the Synopsys Bridge is pre-installed.
-
-# Azure Agent Setup
-
-- Agents can be installed and used on GNU/Linux, macOS, Windows and Docker. See this documentation for details:
-https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=browser
-- You can use Microsoft-hosted agents as well to scan your code using Azure Pipelines.
+|`SYNOPSYS_BRIDGE_PATH`| Provide a path, where you want to configure or already configured Synopsys Bridge.<br/> [Note - If you don't provide any path, then by default configuration path will be considered as - $HOME/synopsys-bridge].<br/> If the configured Synopsys Bridge is not the latest one, latest Synopsys Bridge version will be downloaded          | Optional     |
+| `BRIDGE_DOWNLOAD_URL`      | Provide URL to bridge zip file.<br/> If provided, Synopsys Bridge will be automatically downloaded and configured in the provided bridge- or default- path.<br/> [Note - As per current behavior, when this value is provided, the bridge_path or default path will be cleaned first then download and configured all the time]               | Optional     |
```

Comment:
[Note - As per current behavior, when this value is provided, the bridge_path or default path will be cleaned first then download and configured all the time]            - just confirm it's current behaviour

Reply:
can we say cleared instead of cleaned

Reply:
i think we can remove this part, we have removed this in Gitlab as well